rnsim:
    # ID numbers of the UWB nodes, we are using a convention where the list is
    # supposed to be plit into two rows, first row lists the supposedly static
    # nodes (anchors), and second row is for moving nodes
    nodes_id:
        [100, 101, 102, 103,
         200]
    # 3D position of the nodes in the supposed World coordinates systems, each
    # row should have three coordinates (x, y, z) of the corresponding node on
    # startup. If the node's coordinate is undetermined on startup, put 9999
    # into it.
    nodes_pos:
        [5,  5, 1,
         -5, 5, 0,
         -5, -5, 1,
         5,  -5, 0,
         9999, 9999, 9999
        ]
    # 3D coordinates of the anchors in each node's body frame. Each row starts
    # with the node's ID, then the number of antennas, then the triplets of
    # anchor coordinates. Nodes that don't have anything declared for them are
    # assumed to have a single antenna at its origin
    antennas_pos:
        [200, 2, 0.0, 0.25, 0.0, 0.0, -0.25, 0.0,
         201, 2, 0.0, 0.0, 0.0, 0.0, 0.25, 0.0]
